### PR TITLE
scala-native 0.5

### PR DESCRIPTION
**Pull request by xuwei-k**

### CODE REVIEW

File: project/depends.scala [hunk 1]
```diff
@@ -37,14 +37,13 @@ object depends {
   def nativeTest =
     Seq(libraryDependencies ++= Seq(
     "org.scala-native" %%% "test-interface" % nativeVersion,
-    "org.portable-scala" %%% "portable-scala-reflect" % "1.1.1"
```
Comment (xuwei-k) [inline project/depends.scala:40]: - https://github.com/portable-scala/portable-scala-reflect/pull/45

**Comment by etorreborre** [comment]
Thanks for all those updates @xuwei-k!

**Comment by etorreborre** [comment]
@xuwei-k it is [published now](https://ossindex.sonatype.org/component/pkg:maven/org.specs2/specs2-core_2.12@4.20.6).

File: build.sbt [hunk 2]
```diff
@@ -315,12 +314,14 @@ lazy val shapeless = crossProject(JSPlatform, JVMPlatform, NativePlatform).
...
-  nativeSettings(depends.nativeTest, commonNativeSettings).
+  //nativeSettings(depends.nativeTest, commonNativeSettings).
   dependsOn(matcher, matcherExtra % "test->test")
 
 lazy val shapelessJS = shapeless.js
 lazy val shapelessJVM = shapeless.jvm
-lazy val shapelessNative = shapeless.native
+
+// TODO https://github.com/milessabin/shapeless/issues/1355
+// lazy val shapelessNative = shapeless.native
```
Comment (xuwei-k) [inline build.sbt:324]: - https://github.com/etorreborre/specs2/pull/1242

**Pull request merged**